.gitignore

```diff
@@ -85,3 +85,8 @@ portmaps.json
 apps/testkit.net/testkit.net.userprefs
 apps/testkit.net/Testkit.Net.Desktop/bin/*
 apps/testkit.net/Testkit.Net.Desktop/obj/*
+apps/testkit.java/Testkit.java.android/Tests/AndroidClient2/build
```

Comment:
oops sorry, I overlooked at .gitignore , it is not required as AndroiClient2 is removed now